crowdin/model/webhooks.go

```diff
@@ -0,0 +1,193 @@
+package model
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+)
+
+// Event is a type that represents an event that happens
+// in a Crowdin project.
+type Event string
+
+const (
+	// Project file is added.
+	EventFileAdded Event = "file.added"
```

Comment:
Do we need the `Event` prefix in each of these events? Wouldn't it still be clear what it is without the prefix?